workers/unity/Packages/io.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -174,6 +175,12 @@ public static void Clean()
                 PlayerSettings.SetScriptingBackend(buildTargetGroup, buildContext.ScriptingImplementation);
             }
 
+            var activeIosSdkVersion = PlayerSettings.iOS.sdkVersion;
```

Comment:
now we are using `IOS`, `Ios` and `iOS` xD can we make it a bit more consistent

Reply:
Seems I missed that one and the BuildContext one. Fixing